audiences/lib/audiences/scim.rb

```diff
@@ -11,7 +11,7 @@ module Scim
   module_function
 
     def resources(type:, client: Scim.client, **options)
-      options = defaults.fetch(type, {}).merge(options)
+      options = (defaults[type] || {}).merge(options)
```

Comment:
Is this an extra parentheses?

```suggestion
      options = defaults[type] || {}).merge(options)
```

Reply:
Nop, this is right

Reply:
Thank you for clarifying @xjunior , I didn't see the closing parentheses at first glance which had me very confused 😅